.eslintrc.json

```diff
@@ -37,23 +37,18 @@
         "leadingUnderscore": "allow"
       }
     ],
-    "no-tabs": "error",
+    "no-tabs": [
+      "error",
+      {
+        "allowIndentationTabs": true
+      }
+    ],
```

Comment:
Special case when running the eslint-config-prettier conflict finder script
https://github.com/prettier/eslint-config-prettier#no-tabs